src/sgx/CMakeLists.txt

```diff
@@ -62,14 +62,77 @@ set(FAASM_SGX_WAMR_SRC_UNTRUSTED
         ${PLATFORM_SHARED_SOURCE_UNTRUSTED}
         )
 
+# --------------------------------------------------------
```

Comment:
Re-ordered a bit some of the targets declared here. Not a proper re-factor, but otherwise it was very difficult to track dependencies.

I'd really like to do a more in-depth re-factor of this file though.